cobaya/prior.py

```diff
@@ -793,6 +793,7 @@ def reference(
         ignore_fixed=False,
         warn_if_no_ref=True,
         random_state=None,
+        proposal_scale: dict[str, float | None] | None = None,
```

Comment:
I would rename the kwarg to sth like `override_std`, bc it's more descriptive, at this level it doesn't matter where it comes from (mcmc proposals or any value when calling the method by hand).